projects/elife.yaml

```diff
@@ -430,6 +430,17 @@ journal:
                         # ELB with no active instances
                         503: "/5xx.html"
                     protocol: http
+                origins:
+                    CloudFrontCDNOrigin:
+                        hostname: end2end--journal.elifesciences.org
+                    LogInOrigin:
+                        hostname: end2end--journal.elifesciences.org
+                        pattern: /log-in
+                        headers:
+                            - X-Requested-With
+                            - Host
+                            - CloudFront-Viewer-Country
```

Comment:
Maybe these should inherit?

Reply:
Advantage of inheriting: we avoid specifying these twice. Disadvantage: we cannot remove headers for a custom origin (it could go to a bucket or other static source that doesn't need those, like in the case of `error-pages`). Can probably wait to see if necessary.

Reply:
Shouldn't need to remove, just add the headers at the appropriate level. But yeah, not that important (yet).